### PR TITLE
zboss: version v3.8.0.1+v4.0.0

### PR DESCRIPTION
**Pull request by tomchy**
- ZBOSS sources: cba8c72f3dd3405cc64d418c2a7e18047cb91ea1

Signed-off-by: Tomasz Chyrowicz <tomasz.chyrowicz@nordicsemi.no>

### CODE REVIEW

**Comment by greg-fer** [comment]
Please make sure to include an update to the ZBOSS docs, including the changelog.

File: zboss/doc/zboss_certification.rst [hunk 1]
```diff
@@ -25,4 +25,4 @@ The certification guarantees that a platform or product will work and will inter
 Nordic Semiconductor provides the Zigbee Compliant Platform.
 You can use this plaform as the building block for your Zigbee Certified Product, which is conformant with the ZCL and BDB standard.
 
-The latest certification entries for the combination of the ZBOSS v3.3 stack, the |NCS|, and Nordic Semiconductor's SoCs are available on the CSA's `Zigbee Compliant Platforms page`_.
+The latest certification entries for the combination of the ZBOSS v3 stack, the |NCS|, and Nordic Semiconductor's SoCs are available on the CSA's `Zigbee Compliant Platforms page`_.
```
Comment (greg-fer) [inline zboss/doc/zboss_certification.rst:28]: v3.0?

Comment (tomchy) [reply zboss/doc/zboss_certification.rst:28]: The second digit increases with nearly every release. If you need it, I propose to write `3.x`

**Review by sebastiandraus — APPROVED** [review]

File: zboss/doc/zboss_certification.rst (hunk 1 above)
Comment (tomchy) [reply zboss/doc/zboss_certification.rst]: Changed to: `v3.3 or newer`

**Review by maciejpietras — APPROVED** [review]

**Review by FilipZajdel — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Comment by LuDuda** [comment]
@greg-fer Are we good to merge this, as I see your change request on manifest update in sdk-nrf?

**Comment by sebastiandraus** [comment]
@LuDuda I was told by @greg-fer this morning that we don't have green light yet

**Comment by greg-fer** [comment]
@LuDuda , no, we're not, due to issues with uploading to FTP. An issue has been raised to IT and is awaiting response.

**Pull request merged**